backend/src/main/java/com/bakdata/conquery/sql/conquery/SqlManagedQuery.java

```diff
@@ -77,8 +72,7 @@ public void visit(Consumer<Visitable> visitor) {
 
 	@Override
 	public List<ColumnDescriptor> generateColumnDescriptions() {
-		// todo(tm): This is basically a duplicate from ManagedQuery, but sets the ResultType to String because the SQL connector doesn't convert types for now.
-		// 			 As soon as the connector properly handles types, we can extract this into a helper and use it for both this and ManagedQuery.
+		// todo (ask awildturtok): create class to extract common functionality
```

Comment:
Die `SqlManagedQuery` und die `ManagedQuery` teilen sich die Implementierung, das könnte ich ggf. in das `SingleTableResult` auslagern.